docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+The `browser-list-comma-separated` argument specifies the list of browsers (separated by commas) where tests will be ran.
+
+### Local Browsers
+
+You can specify local browsers by using paths or [aliases](/testcafe/documentation/using-testcafe/common-concepts/browser-aliases). 
+A list of all the available aliases can be obtained by the [--list-browsers](#b) command.
+
+The following example demonstrates how to run a test in several browsers. 
+The browsers are specified differently: one by using the alias, the other by using the path.
+
+```sh
+testcafe chrome,/applications/safari.app /tests/sample-fixture.js
+```
+
+To run tests against **all installed browsers**, use the `all` alias.
+
+```sh
+testcafe all /tests/sample-fixture.js
+```
+
+### <a class="anchor" name="remotes"></a>Remote Browsers
+
+To run tests on a remote browser, specify `remote` as a browser alias. 
+
+```sh
+testcafe remote /tests/sample-fixture.js
+```
+
+If you want to connect multiple browsers,
+prefix alias with the number of browsers you want to connect. For example, if you need to use three remote browsers, specify *3remote*.
+
+```sh
+testcafe 3remote /tests/sample-fixture.js
+```
+
+TestCafe will provide URLs that you need to open in required browsers on your remote device. 
+
+You can also use the [--qr-code](#qr-code) option to display QR-codes that represent the same URLs. 
+Scan the QR-codes by a device on which you are going to test the application.
+As a result, the browsers will be connected to TestCafe and tests will start.
+
+## File Path/Glob Pattern
+
+The `file-or-glob ...` argument specifies files or directories(separated by a space) to run tests from.
+
+For example, this command runs all tests located in the `/my-tests` directory.
+
+```sh
+testcafe ie /my-tests
+```
+
+The following command runs tests from the certain fixture files.
+
+```sh
+testcafe ie /tests/fixture1.js /tests/fixture2.js
+```
+
+You can also use globbing patterns to specify a set of files. 
```

Comment:
> You can also use globbing patterns

Maybe we should add a link to the [node-glob syntax](https://github.com/isaacs/node-glob) page? Like in the [gulp.src](https://github.com/gulpjs/gulp/blob/master/docs/API.md#globs) topic
/cc @inikulin @MargaritaLoseva